src/mcp_atlassian/server.py

```diff
@@ -184,8 +201,13 @@ async def server_lifespan(server: Server) -> AsyncIterator[AppContext]:
             except Exception as e:
                 logger.error(f"Failed to initialize Jira client: {e}", exc_info=True)
 
+        # Get the list of enabled tools
+        enabled_tools = get_enabled_tools()
+        if enabled_tools:
+            logger.debug(f"Filtering tools to: {', '.join(enabled_tools)}")
+
         # Provide context to the application
-        yield AppContext(confluence=confluence, jira=jira)
+        yield AppContext(confluence=confluence, jira=jira, enabled_tools=enabled_tools)
```

Comment:
Adding `enabled_tools` to the `AppContext` is the correct way to pass this configuration through the application lifecycle. Good integration. 👍